From 07059215727054858edba994d2a07ff4e2b7f20f Mon Sep 17 00:00:00 2001
From: mark gross <mark.gross@intel.com>
Date: Fri, 30 Nov 2018 15:41:46 -0800
Subject: [PATCH 1/2] v4.19.5 update fixup smack_lsm.c

---
 security/smack/smack_lsm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/security/smack/smack_lsm.c b/security/smack/smack_lsm.c
index 0107146089f5..fd419cb219ae 100644
--- a/security/smack/smack_lsm.c
+++ b/security/smack/smack_lsm.c
@@ -422,7 +422,7 @@ static int smk_ptrace_rule_check(struct task_struct *tracer,
 
 	rcu_read_lock();
 	tracercred = __task_cred(tracer);
-	tsp = smack_cred(__task_cred(tracer));
+	tsp = smack_cred(tracercred);
 	tracer_known = smk_of_task(tsp);
 
 	if ((mode & PTRACE_MODE_ATTACH) &&
-- 
2.19.1

