From cee5454c8ffc38fac4a4ad4997bade72fd7b7230 Mon Sep 17 00:00:00 2001
From: Kan Liang <kan.liang@linux.intel.com>
Date: Fri, 22 Apr 2022 07:35:10 -0700
Subject: [PATCH 16/21] perf/x86: Add Intel Alder Lake N support

Signed-off-by: Kan Liang <kan.liang@linux.intel.com>
---
 arch/x86/events/intel/core.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/events/intel/core.c b/arch/x86/events/intel/core.c
index 263d74a59451..69d026de1794 100644
--- a/arch/x86/events/intel/core.c
+++ b/arch/x86/events/intel/core.c
@@ -6135,6 +6135,7 @@ __init int intel_pmu_init(void)
 
 	case INTEL_FAM6_ALDERLAKE:
 	case INTEL_FAM6_ALDERLAKE_L:
+	case INTEL_FAM6_ALDERLAKE_N:
 		/*
 		 * Alder Lake has 2 types of CPU, core and atom.
 		 *
-- 
2.27.0

