From fdca976144be488930d7536544f4c4dd3425402b Mon Sep 17 00:00:00 2001
From: Meng Wei <wei.meng@intel.com>
Date: Tue, 27 Nov 2018 19:31:27 +0800
Subject: [PATCH 016/292] media: add member for trace of dual/multi camera for
 PnP KPIs

Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 include/media/videobuf2-v4l2.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/media/videobuf2-v4l2.h b/include/media/videobuf2-v4l2.h
index 727855463838..6b4b0e3e6955 100644
--- a/include/media/videobuf2-v4l2.h
+++ b/include/media/videobuf2-v4l2.h
@@ -47,6 +47,7 @@ struct vb2_v4l2_buffer {
 	__u32			sequence;
 	__s32			request_fd;
 	struct vb2_plane	planes[VB2_MAX_PLANES];
+	__u32			reserved;
 };
 
 /*
-- 
2.21.0

