From e6ce120a8235d3d018b6751f4770d71f99791854 Mon Sep 17 00:00:00 2001
From: "Baoli.Zhang" <baoli.zhang@intel.com>
Date: Mon, 9 Dec 2024 10:34:18 +0800
Subject: [PATCH 18/76] Hardening the kernel for x86/cpu issues

Signed-off-by: Baoli.Zhang <baoli.zhang@intel.com>
---
 Intel/config | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/Intel/config b/Intel/config
index b0a09d9a9980b..983970473fcf7 100644
--- a/Intel/config
+++ b/Intel/config
@@ -463,13 +463,13 @@ CONFIG_X86_PAT=y
 CONFIG_ARCH_USES_PG_UNCACHED=y
 CONFIG_X86_UMIP=y
 CONFIG_CC_HAS_IBT=y
-# CONFIG_X86_KERNEL_IBT is not set
+CONFIG_X86_KERNEL_IBT=y
 CONFIG_X86_INTEL_MEMORY_PROTECTION_KEYS=y
 CONFIG_X86_INTEL_TSX_MODE_OFF=y
 # CONFIG_X86_INTEL_TSX_MODE_ON is not set
 # CONFIG_X86_INTEL_TSX_MODE_AUTO is not set
 CONFIG_X86_SGX=y
-# CONFIG_X86_USER_SHADOW_STACK is not set
+CONFIG_X86_USER_SHADOW_STACK=y
 CONFIG_EFI=y
 CONFIG_EFI_STUB=y
 CONFIG_EFI_HANDOVER_PROTOCOL=y
@@ -533,6 +533,8 @@ CONFIG_CPU_IBRS_ENTRY=y
 CONFIG_CPU_SRSO=y
 # CONFIG_SLS is not set
 # CONFIG_GDS_FORCE_MITIGATION is not set
+CONFIG_MITIGATION_RFDS=y
+CONFIG_MITIGATION_SPECTRE_BHI=y
 CONFIG_ARCH_HAS_ADD_PAGES=y
 
 #
-- 
2.43.0

