From aeb20db50c251b78a03c81be4936ada909123823 Mon Sep 17 00:00:00 2001
From: "Baoli.Zhang" <baoli.zhang@intel.com>
Date: Mon, 9 Dec 2024 10:39:10 +0800
Subject: [PATCH 22/76] Hardening the kernel memory intialization

Signed-off-by: Baoli.Zhang <baoli.zhang@intel.com>
---
 Intel/config | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Intel/config b/Intel/config
index 8ec0790277f1d..09e6fda4ad38e 100644
--- a/Intel/config
+++ b/Intel/config
@@ -1032,7 +1032,7 @@ CONFIG_SLAB_FREELIST_RANDOM=y
 CONFIG_SLAB_FREELIST_HARDENED=y
 # CONFIG_SLUB_STATS is not set
 CONFIG_SLUB_CPU_PARTIAL=y
-# CONFIG_RANDOM_KMALLOC_CACHES is not set
+CONFIG_RANDOM_KMALLOC_CACHES=y
 # end of SLAB allocator options
 
 CONFIG_SHUFFLE_PAGE_ALLOCATOR=y
@@ -7360,7 +7360,7 @@ CONFIG_CC_HAS_AUTO_VAR_INIT_ZERO_BARE=y
 CONFIG_CC_HAS_AUTO_VAR_INIT_ZERO=y
 CONFIG_INIT_STACK_NONE=y
 # CONFIG_INIT_STACK_ALL_PATTERN is not set
-# CONFIG_INIT_STACK_ALL_ZERO is not set
+CONFIG_INIT_STACK_ALL_ZERO=y
 CONFIG_GCC_PLUGIN_STACKLEAK=y
 # CONFIG_STACKLEAK_RUNTIME_DISABLE is not set
 CONFIG_INIT_ON_ALLOC_DEFAULT_ON=y
-- 
2.43.0

