From ee6ae28d643729115739fd54a53190fd11718984 Mon Sep 17 00:00:00 2001
From: "C, Udhayakumar" <udhayakumar.c@intel.com>
Date: Thu, 21 Jan 2021 18:29:43 +0530
Subject: [PATCH 023/109] Disable Manual Mode for TSENS AND ENABLING GOVERNORS

Signed-off-by: C, Udhayakumar <udhayakumar.c@intel.com>
---
 drivers/misc/intel_tsens/keembay_thermal.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/misc/intel_tsens/keembay_thermal.c b/drivers/misc/intel_tsens/keembay_thermal.c
index d6c8fa8fc3aa..f7a619021f1c 100644
--- a/drivers/misc/intel_tsens/keembay_thermal.c
+++ b/drivers/misc/intel_tsens/keembay_thermal.c
@@ -39,9 +39,6 @@ static void kmb_sensor_read_temp(void __iomem *regs_val,
 {
 	int reg_val, kmb_raw_index;
 
-	/* clear the bit of TSENS_EN and re-enable again */
-	iowrite32(0x00, regs_val + AON_TSENS_CFG);
-	iowrite32(CFG_MASK_MANUAL, regs_val + AON_TSENS_CFG);
 	reg_val = ioread32(regs_val + offset);
 	if (reg_val & sample_valid_mask) {
 		reg_val = (reg_val >> bit_shift) & sample_value;
-- 
2.25.1

