From 8aea7804a0ff9208e4bcafaa580860807d099c64 Mon Sep 17 00:00:00 2001
From: "Pan, Kris" <kris.pan@intel.com>
Date: Tue, 10 Dec 2024 08:21:57 +0000
Subject: [PATCH 26/76] Intel/config: disable CONFIG_INFINIBAND_BNXT_RE

Signed-off-by: Pan, Kris <kris.pan@intel.com>
---
 Intel/config | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Intel/config b/Intel/config
index b57000875d47d..1f128a902a06c 100644
--- a/Intel/config
+++ b/Intel/config
@@ -5905,7 +5905,7 @@ CONFIG_INFINIBAND_ON_DEMAND_PAGING=y
 CONFIG_INFINIBAND_ADDR_TRANS=y
 CONFIG_INFINIBAND_ADDR_TRANS_CONFIGFS=y
 CONFIG_INFINIBAND_VIRT_DMA=y
-CONFIG_INFINIBAND_BNXT_RE=m
+# CONFIG_INFINIBAND_BNXT_RE is not set
 # CONFIG_INFINIBAND_CXGB4 is not set
 # CONFIG_INFINIBAND_EFA is not set
 # CONFIG_INFINIBAND_ERDMA is not set
-- 
2.43.0

