From b93e46a0a2d3d5d1b31ef1db2c5e586220ea7391 Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (VMware)" <rostedt@goodmis.org>
Date: Fri, 17 Aug 2018 09:43:41 -0400
Subject: [PATCH 0056/2156] Linux 4.14.60-rt38

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index a3b2408..49bae8d 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt37
+-rt38
-- 
2.7.4

