From d2cab794f25e20ba804ac7d3e22fd853602d1438 Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (VMware)" <rostedt@goodmis.org>
Date: Fri, 17 Aug 2018 09:50:55 -0400
Subject: [PATCH 0057/2156] Linux 4.14.62-rt39

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 49bae8d..5498386 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt38
+-rt39
-- 
2.7.4

