From 75282867c05612b8771fea59c83770d2a2660ddf Mon Sep 17 00:00:00 2001
From: Zhou Furong <furong.zhou@intel.com>
Date: Tue, 8 Feb 2022 10:24:18 +0800
Subject: [PATCH 069/109] [to-be-remove][pinctl] read old device tree node

---
 drivers/pinctrl/pinctrl-keembay.c | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/drivers/pinctrl/pinctrl-keembay.c b/drivers/pinctrl/pinctrl-keembay.c
index 30d6843fca17..5c9d7377a45a 100644
--- a/drivers/pinctrl/pinctrl-keembay.c
+++ b/drivers/pinctrl/pinctrl-keembay.c
@@ -1545,8 +1545,11 @@ static int keembay_pinctrl_reg(struct keembay_pinctrl *kpc,  struct device *dev)
 
 	keembay_pinctrl_desc.pins = keembay_pins;
 	ret = of_property_read_u32(dev->of_node, "ngpios", &kpc->npins);
-	if (ret < 0)
-		return ret;
+	if (ret < 0) {
+		ret = of_property_read_u32(dev->of_node, "num-gpios", &kpc->npins);
+		if (ret < 0)
+			return ret;
+	}
 	keembay_pinctrl_desc.npins = kpc->npins;
 
 	kpc->pctrl = devm_pinctrl_register(kpc->dev, &keembay_pinctrl_desc, kpc);
-- 
2.25.1

