From ce65687084d9820ca1638daab1d15e68578653fa Mon Sep 17 00:00:00 2001
From: Thomas Gleixner <tglx@linutronix.de>
Date: Fri, 8 Jul 2011 20:25:16 +0200
Subject: [PATCH 0261/1132] Add localversion for -RT release

Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
---
 localversion-rt | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
new file mode 100644
index 0000000..6e44e54
--- /dev/null
+++ b/localversion-rt
@@ -0,0 +1 @@
+-rt12
-- 
2.7.4

