From 1cc895b396263cf9d4669e6aca66e53502ce3871 Mon Sep 17 00:00:00 2001
From: Thomas Gleixner <tglx@linutronix.de>
Date: Fri, 8 Jul 2011 20:25:16 +0200
Subject: [PATCH 0262/1418] Add localversion for -RT release

Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
---
 localversion-rt | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
new file mode 100644
index 0000000..1199eba
--- /dev/null
+++ b/localversion-rt
@@ -0,0 +1 @@
+-rt16
-- 
2.7.4

