From 402a91ab953b6606d8d58c727c8d54828cf7fd8a Mon Sep 17 00:00:00 2001
From: Thomas Gleixner <tglx@linutronix.de>
Date: Fri, 8 Jul 2011 20:25:16 +0200
Subject: [PATCH 0263/1301] Add localversion for -RT release

Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
---
 localversion-rt | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
new file mode 100644
index 0000000..08b3e75
--- /dev/null
+++ b/localversion-rt
@@ -0,0 +1 @@
+-rt14
-- 
2.7.4

