From 63b2f12c5e7474fe7f52fa007ed2f85bb1ed30c3 Mon Sep 17 00:00:00 2001
From: Steven Rostedt <rostedt@goodmis.org>
Date: Wed, 27 Mar 2019 10:00:33 -0400
Subject: [PATCH 0264/1374] Linux 4.19.37-rt19 REBASE

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 1199eba..483ad77 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt16
+-rt19
-- 
2.7.4

