From 5591021e0331b5facab175051487b613a88abe6a Mon Sep 17 00:00:00 2001
From: Steven Rostedt <rostedt@goodmis.org>
Date: Wed, 27 Mar 2019 10:00:33 -0400
Subject: [PATCH 0266/1363] Linux 4.19.31-rt18 REBASE

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 1199eba..9e7cd66 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt16
+-rt18
-- 
2.7.4

