From b4f47eb418e5590eeb2295b68b111f23ddab3992 Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (VMware)" <rostedt@goodmis.org>
Date: Fri, 19 Jul 2019 17:46:46 -0400
Subject: [PATCH 0283/1494] Linux 4.19.72-rt25 REBASE

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 1199eba..c5b71f9 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt16
+-rt25
-- 
2.7.4

