From 7c005718e34690dcb3eec30a9e4b9e3920a14064 Mon Sep 17 00:00:00 2001
From: Tom Zanussi <zanussi@kernel.org>
Date: Wed, 6 May 2020 15:31:36 -0500
Subject: [PATCH 0327/1552] Linux 4.19.120-rt52 REBASE

Signed-off-by: Tom Zanussi <zanussi@kernel.org>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 1199eba..66a5ed8 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt16
+-rt52
-- 
2.7.4

