From 7d4b24a0659ad950937af74835e70c2e6c2d377e Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (VMware)" <rostedt@goodmis.org>
Date: Wed, 5 Sep 2018 08:08:53 -0400
Subject: [PATCH 0436/2104] Linux 4.14.71-rt44 REBASE

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 90290c6..ac4d836 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt29
+-rt44
-- 
2.7.4

