From 5e3f37e4e2c6884e6f5d8f387ba49b1726a2f3c6 Mon Sep 17 00:00:00 2001
From: Tom Zanussi <tom.zanussi@linux.intel.com>
Date: Thu, 28 Feb 2019 18:02:43 -0600
Subject: [PATCH 0449/2292] Linux 4.14.106-rt56 REBASE

Signed-off-by: Tom Zanussi <tom.zanussi@linux.intel.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 90290c6..fdb0f88 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt29
+-rt56
-- 
2.7.4

