From 9e983dbbac4f326581ce560eb3de87d2640978d1 Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (VMware)" <rostedt@goodmis.org>
Date: Fri, 4 Jan 2019 20:59:15 -0500
Subject: [PATCH 0450/2278] Linux 4.14.93-rt53 REBASE

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 90290c6..e6c0dc6 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt29
+-rt53
-- 
2.7.4

