From 1fc854bdc415785800f71c5f110b2beed47f2853 Mon Sep 17 00:00:00 2001
From: Tom Zanussi <tom.zanussi@linux.intel.com>
Date: Thu, 4 Apr 2019 15:06:10 -0500
Subject: [PATCH 0459/2302] Linux 4.14.115-rt59 REBASE

Signed-off-by: Tom Zanussi <tom.zanussi@linux.intel.com>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 90290c6..8cdeb45 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt29
+-rt59
-- 
2.7.4

