From 4dd1ede99100ae0ffdb4a32f9a5a197b87113240 Mon Sep 17 00:00:00 2001
From: Tom Zanussi <zanussi@kernel.org>
Date: Wed, 19 Jun 2019 17:43:35 -0500
Subject: [PATCH 0459/2313] Linux 4.14.126-rt62 REBASE

Signed-off-by: Tom Zanussi <zanussi@kernel.org>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 90290c6..40d81d8 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt29
+-rt62
-- 
2.7.4

