From 97843b6edf9a540a1d3a9f1071dbfd27d686c1d4 Mon Sep 17 00:00:00 2001
From: Tom Zanussi <tom.zanussi@linux.intel.com>
Date: Thu, 4 Apr 2019 15:06:10 -0500
Subject: [PATCH 0461/2304] Linux 4.14.109-rt58 REBASE

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 90290c6..f9df2cf 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt29
+-rt58
-- 
2.7.4

