From 7492b529384d0f90e9164d20951aadefcd538f5c Mon Sep 17 00:00:00 2001
From: Tom Zanussi <zanussi@kernel.org>
Date: Fri, 7 Feb 2020 15:17:03 -0600
Subject: [PATCH 0476/2367] Linux 4.14.170-rt74 REBASE

Signed-off-by: Tom Zanussi <zanussi@kernel.org>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 90290c6..7d028f4 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt29
+-rt74
-- 
2.7.4

