From c83ee32a7443a173c9d6075faaf54d64114a0ab8 Mon Sep 17 00:00:00 2001
From: Tom Zanussi <zanussi@kernel.org>
Date: Fri, 23 Aug 2019 13:51:03 -0500
Subject: [PATCH 0477/2345] Linux 4.14.139-rt66 REBASE

Signed-off-by: Tom Zanussi <zanussi@kernel.org>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 90290c6..d42c097 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt29
+-rt66
-- 
2.7.4

