From 891ceed613f95ec9927befc72142168c55e2a8d4 Mon Sep 17 00:00:00 2001
From: Pan Xiuli <xiuli.pan@linux.intel.com>
Date: Wed, 21 Mar 2018 16:38:54 +0800
Subject: [PATCH 0875/1240] ASoC: SOF: Add SOF_IPC_REGION_EXCEPTION

To store panic info in SOF_IPC_REGION_EXCEPTION memory window.

Signed-off-by: Pan Xiuli <xiuli.pan@linux.intel.com>
---
 include/uapi/sound/sof-ipc.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/uapi/sound/sof-ipc.h b/include/uapi/sound/sof-ipc.h
index 1e24f27..6f91b6c 100644
--- a/include/uapi/sound/sof-ipc.h
+++ b/include/uapi/sound/sof-ipc.h
@@ -764,6 +764,7 @@ enum sof_ipc_region {
 	SOF_IPC_REGION_DEBUG,
 	SOF_IPC_REGION_STREAM,
 	SOF_IPC_REGION_REGS,
+	SOF_IPC_REGION_EXCEPTION,
 };
 
 struct sof_ipc_ext_data_hdr {
-- 
2.7.4

