From 0ae197c8934d67e7d79be845ec88320ff7c5f749 Mon Sep 17 00:00:00 2001
From: Zhong Huiping <huipingdx.zhong@intel.com>
Date: Sat, 28 Apr 2018 12:04:43 +0800
Subject: [PATCH 0982/1240] ad5816g: fix build warning

Change-Id: I137145c2f6267f398a5d3492a77490cd843d2922
Signed-off-by: Zhong Huiping <huipingdx.zhong@intel.com>
---
 drivers/media/i2c/ad5816g.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/media/i2c/ad5816g.c b/drivers/media/i2c/ad5816g.c
index 84722c8..8f04b4d 100644
--- a/drivers/media/i2c/ad5816g.c
+++ b/drivers/media/i2c/ad5816g.c
@@ -167,7 +167,6 @@ static int ad5816g_vcm_init(struct v4l2_subdev *sd)
 static int ad5816g_t_focus_vcm(struct v4l2_subdev *sd, s32 val)
 {
 	struct i2c_client *client = v4l2_get_subdevdata(sd);
-	struct ad5816g_device *ad5816g_dev = subdev_to_ad5816g_dev(sd);
 	int ret;
 	u16 data;
 
-- 
2.7.4

