From bc46ec25e614ca7862a6ed5aacee5346e8af940f Mon Sep 17 00:00:00 2001
From: Binbin Wu <binbin.wu@intel.com>
Date: Thu, 17 May 2018 10:18:15 +0800
Subject: [PATCH 1007/1240] acrn: enable acrn only on X86_64 platform

Signed-off-by: Binbin Wu <binbin.wu@intel.com>
Reviewed-by: Zhao, Yakui <yakui.zhao@intel.com>
Reviewed-by: Tian, Kevin <kevin.tian@intel.com>
Tested-by: Tian, Kevin <kevin.tian@intel.com>
---
 arch/x86/acrn/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/x86/acrn/Kconfig b/arch/x86/acrn/Kconfig
index b7cf03d..aa30915a 100644
--- a/arch/x86/acrn/Kconfig
+++ b/arch/x86/acrn/Kconfig
@@ -5,7 +5,7 @@
 
 config ACRN
 	bool "ACRN guest support"
-	depends on X86
+	depends on X86_64
 	depends on PARAVIRT
 	help
 	  This is the Linux ACRN port.  Enabling this will allow the
-- 
2.7.4

