From bb743147e9d5aaad596b98ff4bc033bd019cfb3d Mon Sep 17 00:00:00 2001
From: Xinyun Liu <xinyun.liu@intel.com>
Date: Thu, 24 May 2018 11:52:44 +0800
Subject: [PATCH 1043/1240] drm/i915/gvt: check msg length before use it

Signed-off-by: Xinyun Liu <xinyun.liu@intel.com>
Reviewed-by: Min He <min.he@intel.com>
Acked-by: Zhao Yakui <yakui.zhao@intel.com>
---
 drivers/gpu/drm/i915/gvt/edid.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/gpu/drm/i915/gvt/edid.c b/drivers/gpu/drm/i915/gvt/edid.c
index a61546c..8acf937 100644
--- a/drivers/gpu/drm/i915/gvt/edid.c
+++ b/drivers/gpu/drm/i915/gvt/edid.c
@@ -450,6 +450,9 @@ void intel_gvt_i2c_handle_aux_ch_write(struct intel_vgpu *vgpu,
 	}
 
 	msg_length = AUX_CTL_MSG_LENGTH(value);
+	if (WARN_ON(msg_length <= 0 || msg_length > 20))
+		return;
+
 	for (rxsize = 0; rxsize < msg_length; rxsize += 4)
 		intel_dp_unpack_aux(vgpu_vreg(vgpu, offset + 4 + rxsize),
 				rxbuf + rxsize, msg_length - rxsize);
-- 
2.7.4

