From 0f1b1ba718cd797eb945f4b2a5d3e1dc57e48e80 Mon Sep 17 00:00:00 2001
From: Anil Kumar <anil.kumar.k@intel.com>
Date: Wed, 20 Mar 2019 13:40:50 -0700
Subject: [PATCH 1064/1214] media: pci: Fix compilation bug in ipu-wrapper.c

Signed-off-by: Anil Kumar <anil.kumar.k@intel.com>
---
 drivers/media/pci/intel/ipu-wrapper.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/media/pci/intel/ipu-wrapper.c b/drivers/media/pci/intel/ipu-wrapper.c
index 112ee71..b9520d5 100644
--- a/drivers/media/pci/intel/ipu-wrapper.c
+++ b/drivers/media/pci/intel/ipu-wrapper.c
@@ -508,10 +508,6 @@ void shared_memory_load(unsigned int mmid, u64 addr, void *data, size_t bytes)
 			"access: %s: Enter addr = 0x%lx bytes = 0x%zx\n", __func__,
 			(unsigned long)addr, bytes);
 
-	if (!data && get_mem_sub_system(mmid)) {
-		dev_err(get_mem_sub_system(mmid)->dev,
-			"%s: data ptr is null\n", __func__);
-
 	if (!data) {
 		if (get_mem_sub_system(mmid))
 			dev_err(get_mem_sub_system(mmid)->dev,
-- 
2.7.4

