From f5a308764084b0ed861cecdb35a728d5041f9427 Mon Sep 17 00:00:00 2001
From: Manisha Chinthapally <manisha.chinthapally@intel.com>
Date: Mon, 3 Jun 2019 13:03:32 -0700
Subject: [PATCH 1134/1214] SEP fix for undeclared variable in lwpmudrv.c

Signed-off-by: Manisha Chinthapally <manisha.chinthapally@intel.com>
---
 drivers/platform/x86/sepdk/sep/lwpmudrv.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/platform/x86/sepdk/sep/lwpmudrv.c b/drivers/platform/x86/sepdk/sep/lwpmudrv.c
index d18bfb6..5a78add 100644
--- a/drivers/platform/x86/sepdk/sep/lwpmudrv.c
+++ b/drivers/platform/x86/sepdk/sep/lwpmudrv.c
@@ -3237,7 +3237,7 @@ static OS_STATUS lwpmudrv_Read_Counters_And_Switch_Group(IOCTL_ARGS arg)
 		// step 2
 		// if per_cpu_tsc is not defined, read cpu0's tsc and save in var cpu_tsc[0]
 		// if per_cpu_tsc is defined, read all cpu's tsc and save in var cpu_tsc by lwpmudrv_Fill_TSC_Info
-#if !defined(CONFIG_PREEMPT_COUNT)
+#if !defined(CONFIG_PREEMPT_COUNT) && !defined(DRV_SEP_ACRN_ON)
 	if (DRV_CONFIG_per_cpu_tsc(drv_cfg)) {
 		atomic_set(&read_now, GLOBAL_STATE_num_cpus(driver_state));
 		init_waitqueue_head(&read_tsc_now);
@@ -3306,7 +3306,7 @@ static OS_STATUS lwpmudrv_Read_Counters_And_Switch_Group(IOCTL_ARGS arg)
 
 	// step 9
 	// if per_cpu_tsc is defined, read all cpu's tsc and save in cpu_tsc for next run
-#if !defined(CONFIG_PREEMPT_COUNT)
+#if !defined(CONFIG_PREEMPT_COUNT) && !defined(DRV_SEP_ACRN_ON)
 	if (DRV_CONFIG_per_cpu_tsc(drv_cfg)) {
 		atomic_set(&read_now, GLOBAL_STATE_num_cpus(driver_state));
 		init_waitqueue_head(&read_tsc_now);
@@ -3393,7 +3393,7 @@ static OS_STATUS lwpmudrv_Read_And_Reset_Counters(IOCTL_ARGS arg)
 		// step 2
 		// if per_cpu_tsc is not defined, read cpu0's tsc into var cpu_tsc[0]
 		// if per_cpu_tsc is defined, read all cpu's tsc into var cpu_tsc by lwpmudrv_Fill_TSC_Info
-#if !defined(CONFIG_PREEMPT_COUNT)
+#if !defined(CONFIG_PREEMPT_COUNT) && !defined(DRV_SEP_ACRN_ON)
 	if (DRV_CONFIG_per_cpu_tsc(drv_cfg)) {
 		atomic_set(&read_now, GLOBAL_STATE_num_cpus(driver_state));
 		init_waitqueue_head(&read_tsc_now);
@@ -3459,7 +3459,7 @@ static OS_STATUS lwpmudrv_Read_And_Reset_Counters(IOCTL_ARGS arg)
 
 	// step 8
 	// if per_cpu_tsc is defined, read all cpu's tsc and save in cpu_tsc for next run
-#if !defined(CONFIG_PREEMPT_COUNT)
+#if !defined(CONFIG_PREEMPT_COUNT) && !defined(DRV_SEP_ACRN_ON)
 	if (DRV_CONFIG_per_cpu_tsc(drv_cfg)) {
 		atomic_set(&read_now, GLOBAL_STATE_num_cpus(driver_state));
 		init_waitqueue_head(&read_tsc_now);
-- 
2.7.4

