From 0e99b368efa1a4c2762e9b6c3c87b26212c5bfde Mon Sep 17 00:00:00 2001
From: "Dutta, Ranjan" <ranjan.dutta@intel.com>
Date: Thu, 23 Jan 2020 11:44:19 +0530
Subject: [PATCH 1224/1225] Fix merge issues for rt38.

---
 drivers/char/random.c | 1 -
 1 file changed, 1 deletion(-)

Index: b/drivers/char/random.c
===================================================================
--- a/drivers/char/random.c
+++ b/drivers/char/random.c
@@ -50,7 +50,6 @@
 #include <linux/syscalls.h>
 #include <linux/completion.h>
 #include <linux/uuid.h>
-#include <linux/locallock.h>
 #include <linux/uaccess.h>
 #include <linux/siphash.h>
 #include <linux/uio.h>
