From 5c6f20912df1ea25bc185e7eba4d477a0cad06f5 Mon Sep 17 00:00:00 2001
From: Liam Girdwood <liam.r.girdwood@linux.intel.com>
Date: Sun, 17 Jun 2018 22:07:50 +0100
Subject: [PATCH 1232/1240] ASoC: SOF: nocodec: give DAI driver more meaningful
 name

Signed-off-by: Liam Girdwood <liam.r.girdwood@linux.intel.com>
---
 sound/soc/sof/nocodec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/sof/nocodec.c b/sound/soc/sof/nocodec.c
index 0685d29..dab4e2e 100644
--- a/sound/soc/sof/nocodec.c
+++ b/sound/soc/sof/nocodec.c
@@ -63,7 +63,7 @@ static struct snd_soc_dai_link sof_nocodec_dais[] = {
 		.name = "NoCodec",
 		.id = 0,
 		.init = nocodec_rtd_init,
-		.cpu_dai_name = "sof-audio",
+		.cpu_dai_name = "sof-nocodec-dai",
 		.platform_name = "sof-audio",
 		.no_pcm = 1,
 		.codec_dai_name = "snd-soc-dummy-dai",
-- 
2.7.4

