From b8a37f48421f57ae1904550a9cd5fc7b25e02410 Mon Sep 17 00:00:00 2001
From: Liam Girdwood <liam.r.girdwood@linux.intel.com>
Date: Sun, 17 Jun 2018 22:07:54 +0100
Subject: [PATCH 1236/1240] ASoC: SOF: Intel HSW platform DAIs

Signed-off-by: Liam Girdwood <liam.r.girdwood@linux.intel.com>
Signed-off-by: Keyon Jie <yang.jie@linux.intel.com>
---
 sound/soc/sof/intel/hsw.c | 29 +++++++++++++++++++++++++++++
 1 file changed, 29 insertions(+)

diff --git a/sound/soc/sof/intel/hsw.c b/sound/soc/sof/intel/hsw.c
index e401673..5e2c50b 100644
--- a/sound/soc/sof/intel/hsw.c
+++ b/sound/soc/sof/intel/hsw.c
@@ -709,6 +709,32 @@ static int hsw_remove(struct snd_sof_dev *sdev)
 	return 0;
 }
 
+#define HSW_FORMATS (SNDRV_PCM_FMTBIT_S16_LE | SNDRV_PCM_FMTBIT_S24_LE | \
+	SNDRV_PCM_FMTBIT_S32_LE)
+
+/* Haswell DAIs */
+static struct snd_soc_dai_driver hsw_dai[] = {
+{
+	.name = "ssp0-port",
+	.playback = SOF_DAI_STREAM("ssp0 Tx", 1, 8,
+				   SNDRV_PCM_RATE_8000_192000, HSW_FORMATS),
+	.capture = SOF_DAI_STREAM("ssp0 Rx", 1, 8,
+				  SNDRV_PCM_RATE_8000_192000, HSW_FORMATS),
+},
+{
+	.name = "ssp1-port",
+	.playback = SOF_DAI_STREAM("ssp1 Tx", 1, 8,
+				   SNDRV_PCM_RATE_8000_192000, HSW_FORMATS),
+	.capture = SOF_DAI_STREAM("ssp1 Rx", 1, 8,
+				  SNDRV_PCM_RATE_8000_192000, HSW_FORMATS),
+},
+};
+
+struct snd_sof_dai_drv hsw_dai_drv = {
+	.drv = hsw_dai,
+	.num_drv = ARRAY_SIZE(hsw_dai)
+};
+
 /* haswell ops */
 struct snd_sof_dsp_ops sof_hsw_ops = {
 	/*Device init */
@@ -751,6 +777,9 @@ struct snd_sof_dsp_ops sof_hsw_ops = {
 	/*Firmware loading */
 	.load_firmware	= snd_sof_load_firmware_memcpy,
 
+	/* DAI drivers */
+	.dai_drv = &hsw_dai_drv,
+
 };
 EXPORT_SYMBOL(sof_hsw_ops);
 
-- 
2.7.4

