From 27cbcbb546ef146952149606f5bda076e192d113 Mon Sep 17 00:00:00 2001
From: Liam Girdwood <liam.r.girdwood@linux.intel.com>
Date: Wed, 18 Jul 2018 20:53:49 +0100
Subject: [PATCH 1342/1619] ASoC: SOF: topology: Add comments to help
 readability.

Signed-off-by: Liam Girdwood <liam.r.girdwood@linux.intel.com>
---
 sound/soc/sof/topology.c | 19 ++++++++++++++++++-
 1 file changed, 18 insertions(+), 1 deletion(-)

diff --git a/sound/soc/sof/topology.c b/sound/soc/sof/topology.c
index 143a191..7907199 100644
--- a/sound/soc/sof/topology.c
+++ b/sound/soc/sof/topology.c
@@ -204,6 +204,10 @@ static enum sof_ipc_dai_type find_dai(const char *name)
 	return SOF_DAI_INTEL_NONE;
 }
 
+/*
+ * Supported Frame format types and lookup, add new ones to end of list.
+ */
+
 struct sof_frame_types {
 	const char *name;
 	enum sof_ipc_frame frame;
@@ -265,6 +269,11 @@ static int sof_control_load_volume(struct snd_soc_component *scomp,
 	return 0;
 }
 
+/*
+ * Topology Token Parsing.
+ * New tokens should be added to headers and parsing tables below.
+ */
+
 struct sof_topology_token {
 	u32 token;
 	u32 type;
@@ -1218,6 +1227,7 @@ static int sof_widget_load(struct snd_soc_component *scomp, int index,
 			   struct snd_soc_dapm_widget *w,
 			   struct snd_soc_tplg_dapm_widget *tw)
 {
+	/* nothing todo atm */
 	return 0;
 }
 
@@ -1377,6 +1387,10 @@ static int sof_widget_unload(struct snd_soc_component *scomp,
 	return 0;
 }
 
+/*
+ * DAI HW configuration.
+ */
+
 /* FE DAI - used for any driver specific init */
 static int sof_dai_load(struct snd_soc_component *scomp, int index,
 			struct snd_soc_dai_driver *dai_drv,
@@ -1491,7 +1505,8 @@ static int sof_link_ssp_load(struct snd_soc_component *scomp, int index,
 		config->id, config->format,
 		config->ssp.mclk_rate, config->ssp.bclk_rate,
 		config->ssp.fsync_rate, config->ssp.sample_valid_bits,
-		config->ssp.tdm_slot_width, config->ssp.tdm_slots, config->ssp.mclk_id);
+		config->ssp.tdm_slot_width, config->ssp.tdm_slots,
+		config->ssp.mclk_id);
 
 	/* send message to DSP */
 	ret = sof_ipc_tx_message(sdev->ipc,
@@ -1860,6 +1875,7 @@ static int sof_route_load(struct snd_soc_component *scomp, int index,
 static int sof_route_unload(struct snd_soc_component *scomp,
 			    struct snd_soc_dobj *dobj)
 {
+	/* TODO: unload routes when topology is changed */
 	return 0;
 }
 
@@ -1913,6 +1929,7 @@ static void sof_complete(struct snd_soc_component *scomp)
 static int sof_manifest(struct snd_soc_component *scomp, int index,
 			struct snd_soc_tplg_manifest *man)
 {
+	/* not currently parsed */
 	return 0;
 }
 
-- 
2.7.4

