From 9659983642421a69ef282ff370b9eb1e9423dde6 Mon Sep 17 00:00:00 2001
From: Liam Girdwood <liam.r.girdwood@linux.intel.com>
Date: Thu, 19 Jul 2018 20:03:01 +0100
Subject: [PATCH 1351/1606] ASoC: SOF: pcm: cleanups for upstream

Checkpatch cleanups.

Signed-off-by: Liam Girdwood <liam.r.girdwood@linux.intel.com>
---
 sound/soc/sof/pcm.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/sound/soc/sof/pcm.c b/sound/soc/sof/pcm.c
index 92da307..e4048f1 100644
--- a/sound/soc/sof/pcm.c
+++ b/sound/soc/sof/pcm.c
@@ -141,7 +141,6 @@ static int sof_pcm_hw_params(struct snd_pcm_substream *substream,
 							       params);
 	dev_dbg(sdev->dev, "stream_tag %d", pcm.params.stream_tag);
 
-
 	/* send IPC to the DSP */
 	ret = sof_ipc_tx_message(sdev->ipc, pcm.hdr.cmd, &pcm, sizeof(pcm),
 				 &ipc_params_reply, sizeof(ipc_params_reply));
@@ -233,7 +232,6 @@ static int sof_pcm_trigger(struct snd_pcm_substream *substream, int cmd)
 		return -EINVAL;
 	}
 
-	/* set RUN firstly per the sequence suggested by firmware team */
 	snd_sof_pcm_platform_trigger(sdev, substream, cmd);
 
 	/* send IPC to the DSP */
-- 
2.7.4

