From 0000db153c6331697e9eef0b49ac122c8248ebbb Mon Sep 17 00:00:00 2001
From: Rander Wang <rander.wang@linux.intel.com>
Date: Fri, 27 Jul 2018 16:31:46 +0800
Subject: [PATCH 1366/1670] ASoC: SOF: fix a typo

Signed-off-by: Rander Wang <rander.wang@linux.intel.com>
---
 sound/soc/sof/ipc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/sof/ipc.c b/sound/soc/sof/ipc.c
index be6a98e..150fc4e 100644
--- a/sound/soc/sof/ipc.c
+++ b/sound/soc/sof/ipc.c
@@ -182,7 +182,7 @@ static void ipc_tx_next_msg(struct work_struct *work)
 	if (list_empty(&ipc->tx_list) || !snd_sof_dsp_is_ready(sdev))
 		goto out;
 
-	/* sned first message in TX list */
+	/* send first message in TX list */
 	msg = list_first_entry(&ipc->tx_list, struct snd_sof_ipc_msg, list);
 	list_move(&msg->list, &ipc->reply_list);
 	snd_sof_dsp_send_msg(sdev, msg);
-- 
2.7.4

