From 0265b1656f3974775756e1af05e0d61367958873 Mon Sep 17 00:00:00 2001
From: "Dutta, Ranjan" <ranjan.dutta@intel.com>
Date: Thu, 23 Jan 2020 11:44:19 +0530
Subject: [PATCH 1505/1505] Fix merge issues for rt38.

---
 drivers/char/random.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/char/random.c b/drivers/char/random.c
index a4db380eb87e..468a128c8ad8 100644
--- a/drivers/char/random.c
+++ b/drivers/char/random.c
@@ -265,7 +265,6 @@
 #include <linux/syscalls.h>
 #include <linux/completion.h>
 #include <linux/uuid.h>
-#include <linux/locallock.h>
 #include <crypto/chacha20.h>
 
 #include <asm/processor.h>
-- 
2.17.1

