From 875e05e7bfe537db7dc5491de4df1b4156ec2bb4 Mon Sep 17 00:00:00 2001
From: Andrew kim <andrew.kim@intel.com>
Date: Sun, 28 Jul 2019 18:59:20 -0700
Subject: [PATCH 1546/1548] fix merge issue for RDT update in rt24

Signed-off-by: Andrew kim <andrew.kim@intel.com>
---
 arch/x86/kernel/cpu/resctrl/rdtgroup.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/x86/kernel/cpu/resctrl/rdtgroup.c b/arch/x86/kernel/cpu/resctrl/rdtgroup.c
index 7ca5e3b358e0..aa1a6e68f318 100644
--- a/arch/x86/kernel/cpu/resctrl/rdtgroup.c
+++ b/arch/x86/kernel/cpu/resctrl/rdtgroup.c
@@ -2514,11 +2514,7 @@ static int rdtgroup_init_alloc(struct rdtgroup *rdtgrp)
 				if (closid_allocated(i) && i != closid) {
 					mode = rdtgroup_mode_by_closid(i);
 					if (mode == RDT_MODE_PSEUDO_LOCKSETUP)
-<<<<<<< HEAD
 						continue;
-					used_b |= *ctrl;
-=======
-						break;
 					/*
 					 * If CDP is active include peer
 					 * domain's usage to ensure there
@@ -2530,7 +2526,6 @@ static int rdtgroup_init_alloc(struct rdtgroup *rdtgrp)
 					else
 						peer_ctl = 0;
 					used_b |= *ctrl | peer_ctl;
->>>>>>> dc5c93e... x86/intel_rdt: Fix initial allocation to consider CDP
 					if (mode == RDT_MODE_SHAREABLE)
 						d->new_ctrl |= *ctrl | peer_ctl;
 				}
-- 
2.20.1

