From e98d16dc1b8d1151cd94af4834c318588c5c9718 Mon Sep 17 00:00:00 2001
From: "Dutta, Ranjan" <ranjan.dutta@intel.com>
Date: Thu, 23 Jan 2020 11:44:19 +0530
Subject: [PATCH 1547/1548] Fix merge issues for rt38.

---
 drivers/char/random.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/char/random.c b/drivers/char/random.c
index e1617ecfca96..d15dd08a92e2 100644
--- a/drivers/char/random.c
+++ b/drivers/char/random.c
@@ -265,7 +265,6 @@
 #include <linux/syscalls.h>
 #include <linux/completion.h>
 #include <linux/uuid.h>
-#include <linux/locallock.h>
 #include <crypto/chacha20.h>
 
 #include <asm/processor.h>
-- 
2.20.1

